### ISSUE TITLE
Hyperlink function in sheets

### ISSUE THREAD

**Issue by NanaChabe** [infojunkie/socialcalc#11]
Hi

Using this librarie for Drupal Sheetnode i detect a problem with links.

In the opction of import an xlsx and save information, the function hyperlink is unknown, How can i make this work? there is a compatibility for this?

Thanks

**Comment by infojunkie** [comment]
Can you explain more what you mean by

> the function hyperlink is unknown

What fails and at which stage (during import, during display, etc.) ?

**Comment by NanaChabe** [comment]
When i upload a file to import a sheet in drupal cms, in the display result, the table with cells that containt hyperlink show the message "unknown function hyperlink".

for example: for join information the function is "=CONCATENATE" but in drupal sheetnode module the result mesage is "unknown function concatenate" after a searching i found that concatenate function can be changed for "&" caracter, and now i can import the information  without problem.

But this not happen in hypelink function, there is just a error

Hope be clear

**Comment by infojunkie** [comment]
Yes, this function is not implemented in SocialCalc. To make it work, you need to implement this JavaScript function yourself. You can see an example at http://cgit.drupalcode.org/sheetnode/tree/js/sheetnode.js#n33 (of course, your function does not need to make an AJAX call - it just needs to format an anchor tag).

You can add this function to `sheetnode.js`, or you can create your own Drupal module that implements `hook_sheetnode_plugins` to declare your own JS file that should be loaded when the spreadsheet is loaded.
